src/theme/types.ts

```diff
@@ -47,3 +47,8 @@ declare module 'styled-components' {
     mq: DefaultTheme['mediaQueries']
   }
 }
+
+export enum AnalyticsDimension {
+  NETWORK,
+  BROWSER_TYPE,
+}
```

Comment:
Why is this on theme?

Reply:
Good catch 🙈

Reply:
https://github.com/gnosis/gp-ui/pull/322